assets/index.css

```diff
@@ -234,25 +239,23 @@ main>section:not(:first-child) {
   max-width: 840px;
 }
 
-.about__header,
-.features__header,
-.links__header {
-  font-weight: 500;
-}
-
 @media screen and (prefers-color-scheme: dark) {
   .hero__content {
     color: var(--text-color);
   }
 
   .hero__content__icon {
-    filter: brightness(75%);
+    filter: brightness(90%);
   }
 
   .hero__content__description {
     margin-bottom: 30px;
   }
 
+  .about__info {
+    width: 100%;
+  }
+
```

Comment:
Why should the about info be 100% on dark mode?

Also I may have misplaced the
```css
  .hero__content__description {
    margin-bottom: 30px;
  }
```

text. It should be in the `@media max-width: 800` (or 520?) part.